src/components/pages/TeacherDashboard/TeacherViewRumble/RumbleStudentList/RenderRumbleStudentList.tsx

```diff
@@ -1,31 +1,53 @@
 import React from 'react';
+import { useSetRecoilState } from 'recoil';
 import { Sections, Students } from '../../../../../api';
+import noStudents from '../../../../../assets/img/no-students.svg';
+import { modals } from '../../../../../state';
 import { Table } from '../../../../common';
+import { InviteToSection } from '../../InviteToSection';
 import RumbleStudentCard from './RumbleStudentCard';
 
 const RenderRumbleStudentList = ({
   studentList,
   section,
 }: IRenderRumbleStudentListProps): React.ReactElement => {
+  const setInviteModalOpen = useSetRecoilState(modals.invite.isOpen);
+  const openInviteModal = () => setInviteModalOpen(true);
+
   return (
     <div className="rumble-student-list-wrapper">
       <div className="rumble-student-list-container">
         <div className="rumble-student-list">
-          <Table.Header>
-            <Table.Col>Student Name</Table.Col>
-            <Table.Col>Last Name</Table.Col>
-            <Table.Col>First Name</Table.Col>
-            <Table.Col>Submission Status</Table.Col>
-          </Table.Header>
-          <Table.Body>
-            {studentList.map((student) => (
-              <RumbleStudentCard
-                student={student}
-                section={section}
-                key={student.id}
-              />
-            ))}
-          </Table.Body>
+          <InviteToSection />
```

Comment:
I think the modal is already being displayed somewhere else and we don't need to put in in again, that's what the recoil state is for